dora/core/client/fs/src/test/java/alluxio/client/file/MetadataCachingFileSystemTest.java

```diff
@@ -73,6 +73,7 @@ public void before() throws Exception {
     mConf.set(PropertyKey.USER_METADATA_CACHE_MAX_SIZE, 1000, Source.RUNTIME);
     // Avoid async update file access time to call getStatus to mess up the test results
     mConf.set(PropertyKey.USER_UPDATE_FILE_ACCESSTIME_DISABLED, true);
+    mConf.set(PropertyKey.USER_METADATA_CACHE_NOT_FOUND_CACHE_ENABLED, true);
```

Comment:
Since the default value for `USER_METADATA_CACHE_NOT_FOUND_CACHE_ENABLED` is `false`, add a unit test to test that when it's not enabled, a missing file won't be cached.